### DISCUSSION TITLE
Trying to run docker image in kubernetes

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2782] [Archive]
I try to start docker image using kubernetes and get this after start:

```
[Tue Jul 07 15:52:05.817415 2020] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) PHP/7.3.19 configured -- resuming normal operations
[Tue Jul 07 15:52:05.817542 2020] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
10.244.3.9 - - [07/Jul/2020:15:54:49 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:49 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:49 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:50 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:50 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:50 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:50 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:50 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:50 +0000] "\x16\x03\x01" 400 0 "-" "-"
10.244.3.9 - - [07/Jul/2020:15:54:50 +0000] "\x16\x03\x01" 400 0 "-" "-"
```

and seems is not accessible the UI, any ideas?

I did setup mysql and db/user freshly, I guess ti creates schema by default?

**Comment by botzill** [comment]
OK, now I see this:

<img width="612" alt="Screen Shot 2020-07-07 at 19 24 08" src="https://user-images.githubusercontent.com/4686040/86812317-8e601180-c087-11ea-9885-c302075760ee.png">

How can I create this tables, are not created automatically?

**Reply by WoLfulus** [reply]
You'll need to run a task within your cluster to execute the database installation. That installs the database e runs the migrations.

The command should be the equivalent of https://docs.directus.io/installation/docker.html#step-4-initialize-the-database-and-an-admin-user

**Reply by botzill** [reply]
I see, but why we can't do that automatically? Like to have some kind of init script into docker container that we can give as input?
